guidelines.md

```diff
@@ -220,4 +171,5 @@ It's recommended that 2 developers review each major pull request though there's
 1. Your component should meet all of the above and align with our UI/UX [goals](#ui--ux-goals)
 1. Tests should pass - `npm test`
 1. Lint checking should pass - `npm run lint`
+1. Format checking should pass - `npx prettier --check ./src/**/*.js ./.storybook/**/*.js ./stories/**/*.js`
```

Comment:
I believe this can just be `npm run format` or maybe we could add a script in `package.json` that only does the `--check` flag

Reply:
`npm run format` formats the whole project which makes it little intrusive. I will probably add a check task.